source/content/traffic-limits.md

```diff
@@ -57,10 +57,13 @@ Static requests (images PDFs, CSS, JS, etc) are not included in our normal traff
 
 However we do reserve the right to review individual sites that are excessive bandwidth consumers. If sites are serving static assets at an excessive rate, this can be considered plan abuse.
 
+### What about 404s and other "client errors?"
+Pantheon only counts pages returned, considered [`200` level](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#2xx_Success), in traffic metrics. The Platform does not count "client errors," which are returned as [`400` level](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#4xx_Client_errors), as part of plan traffic limits.
```

Comment:
@alexfornuto
I added this after a discussion on Slack. if the copy is good, I think we're good to merge this PR

Reply:
Yup, just waiting to see the preview.